## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 433497`

Logs: https://github.com/GaetanLepage/nixpkgs-review-gha/actions/runs/16949859344


---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.finetuning-scheduler</summary>
<pre>INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
===== 1 failed, 128 passed, 56 skipped, 510 warnings in 249.06s (0:04:09) ======

Epoch 7: 100%|██████████| 64/64 [00:18<00:00,  3.50it/s, v_num=0] </pre>
</details>
<details>
<summary>python313Packages.finetuning-scheduler</summary>
<pre>/nix/store/39qfvci8jdb0af7wdh5rxs5amnnnml4z-python3.13-finetuning-scheduler-2.5.1/lib/python3.13/site-packages/finetuning_scheduler/fts_supporters.py:154: IndexError
----------------------------- Captured stderr call -----------------------------
INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
= 1 failed, 126 passed, 56 skipped, 2 deselected, 499 warnings in 293.17s (0:04:53) =</pre>
</details>
</details>


---
### `aarch64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>python312Packages.finetuning-scheduler</summary>
<pre>INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
= 1 failed, 126 passed, 56 skipped, 2 deselected, 496 warnings in 196.18s (0:03:16) =

Epoch 7: 100%|██████████| 64/64 [00:10<00:00,  6.07it/s, v_num=0] </pre>
</details>
<details>
<summary>python313Packages.finetuning-scheduler</summary>
<pre>/nix/store/wskp3dsb99af3c6x4hi9awqmv4h4wlc0-python3.13-finetuning-scheduler-2.5.1/lib/python3.13/site-packages/finetuning_scheduler/fts_supporters.py:154: IndexError
----------------------------- Captured stderr call -----------------------------
INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/build/pytest-of-nixbld/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
= 1 failed, 124 passed, 56 skipped, 4 deselected, 485 warnings in 212.44s (0:03:32) =</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python312Packages.finetuning-scheduler</summary>
<pre>/nix/store/nhdviyk7xpcg8qzpnwniwb5lkgxy19m7-python3.12-finetuning-scheduler-2.5.1/lib/python3.12/site-packages/finetuning_scheduler/fts_supporters.py:154: IndexError
----------------------------- Captured stderr call -----------------------------
INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-5662-1325994810/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-5662-1325994810/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-5662-1325994810/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-5662-1325994810/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
===== 1 failed, 126 passed, 58 skipped, 499 warnings in 285.56s (0:04:45) ======</pre>
</details>
<details>
<summary>python313Packages.finetuning-scheduler</summary>
<pre>/nix/store/lg5b15gs5xmp8kxmqmn1mam2g5ryaadh-python3.13-finetuning-scheduler-2.5.1/lib/python3.13/site-packages/finetuning_scheduler/fts_supporters.py:154: IndexError
----------------------------- Captured stderr call -----------------------------
INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-5662-1325994817/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-5662-1325994817/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-5662-1325994817/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-5662-1325994817/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
= 1 failed, 126 passed, 56 skipped, 2 deselected, 499 warnings in 240.49s (0:04:00) =</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.finetuning-scheduler</li>
    <li>python312Packages.finetuning-scheduler.dist</li>
    <li>python313Packages.finetuning-scheduler</li>
    <li>python313Packages.finetuning-scheduler.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>python312Packages.fastmri</li>
    <li>python312Packages.fastmri.dist</li>
    <li>python312Packages.gluonts</li>
    <li>python312Packages.gluonts.dist</li>
    <li>python312Packages.lightning</li>
    <li>python312Packages.lightning.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-lightning</li>
    <li>python312Packages.pytorch-lightning.dist</li>
    <li>python312Packages.whisperx</li>
    <li>python312Packages.whisperx.dist</li>
    <li>python313Packages.fastmri</li>
    <li>python313Packages.fastmri.dist</li>
    <li>python313Packages.gluonts</li>
    <li>python313Packages.gluonts.dist</li>
    <li>python313Packages.lightning</li>
    <li>python313Packages.lightning.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pytorch-lightning</li>
    <li>python313Packages.pytorch-lightning.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.finetuning-scheduler</summary>
<pre>/nix/store/5xqrqjs19nmlagwjpr39bd698ls9bsm5-python3.12-finetuning-scheduler-2.5.1/lib/python3.12/site-packages/finetuning_scheduler/fts_supporters.py:154: IndexError
----------------------------- Captured stderr call -----------------------------
INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-4916-3081909964/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-4916-3081909964/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-4916-3081909964/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/nix/build/nix-build-python3.12-finetuning-scheduler-2.5.1.drv-4916-3081909964/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
===== 1 failed, 126 passed, 58 skipped, 499 warnings in 132.64s (0:02:12) ======</pre>
</details>
<details>
<summary>python313Packages.finetuning-scheduler</summary>
<pre>/nix/store/vwzm9j89ymw35jbsb7zwdpgma176g794-python3.13-finetuning-scheduler-2.5.1/lib/python3.13/site-packages/finetuning_scheduler/fts_supporters.py:154: IndexError
----------------------------- Captured stderr call -----------------------------
INFO:pytorch_lightning.utilities.rank_zero:💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO:pytorch_lightning.utilities.rank_zero:GPU available: False, used: False
INFO:pytorch_lightning.utilities.rank_zero:TPU available: False, using: 0 TPU cores
INFO:pytorch_lightning.utilities.rank_zero:HPU available: False, using: 0 HPUs
INFO:pytorch_lightning.utilities.rank_zero:No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO:pytorch_lightning.utilities.rank_zero:fine-tuning schedule dumped to /nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-4916-3081909971/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO:pytorch_lightning.utilities.rank_zero:Generated default fine-tuning schedule '/nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-4916-3081909971/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
------------------------------ Captured log call -------------------------------
INFO     pytorch_lightning.utilities.rank_zero:callback_connector.py:108 💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
INFO     pytorch_lightning.utilities.rank_zero:setup.py:156 GPU available: False, used: False
INFO     pytorch_lightning.utilities.rank_zero:setup.py:159 TPU available: False, using: 0 TPU cores
INFO     pytorch_lightning.utilities.rank_zero:setup.py:169 HPU available: False, using: 0 HPUs
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1332 No fine-tuning schedule provided, max_depth set to -1 so iteratively thawing entire model
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1421 fine-tuning schedule dumped to /nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-4916-3081909971/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml.
INFO     pytorch_lightning.utilities.rank_zero:fts_supporters.py:1396 Generated default fine-tuning schedule '/nix/build/nix-build-python3.13-finetuning-scheduler-2.5.1.drv-4916-3081909971/pytest-of-_nixbld3/pytest-0/test_fts_misconfiguration_no_m0/lightning_logs/version_0/FinetuningSchedulerBoringModel_ft_schedule.yaml' for iterative fine-tuning
=========================== short test summary info ============================
FAILED tests/test_finetuning_scheduler_callback.py::test_fts_misconfiguration[no_multi_targ] - IndexError: list index out of range
= 1 failed, 126 passed, 56 skipped, 2 deselected, 499 warnings in 144.85s (0:02:24) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/finetuning-scheduler/default.nix
+++ b/pkgs/development/python-modules/finetuning-scheduler/default.nix
@@ -58,23 +58,28 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [ pytestCheckHook ];
   enabledTestPaths = [ "tests" ];
-  disabledTests =
-    lib.optionals (pythonOlder "3.12") [
-      # torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised:
-      # LoweringException: ImportError: cannot import name 'triton_key' from 'triton.compiler.compiler'
-      "test_fts_dynamo_enforce_p0"
-      "test_fts_dynamo_resume"
-      "test_fts_dynamo_intrafit"
-    ]
-    ++ lib.optionals (pythonAtLeast "3.13") [
-      # RuntimeError: Dynamo is not supported on Python 3.13+
-      "test_fts_dynamo_enforce_p0"
-      "test_fts_dynamo_resume"
-    ]
-    ++ lib.optionals (stdenv.hostPlatform.isAarch64 && stdenv.hostPlatform.isLinux) [
-      # slightly exceeds numerical tolerance on aarch64-linux:
-      "test_fts_frozen_bn_track_running_stats"
-    ];
+  disabledTests = [
+    # Fails since pytorch-lightning was bumped to 2.5.3
+    # IndexError: list index out of range
+    # https://github.com/speediedan/finetuning-scheduler/issues/19
+    "test_fts_misconfiguration"
+  ]
+  ++ lib.optionals (pythonOlder "3.12") [
+    # torch._dynamo.exc.BackendCompilerFailed: backend='inductor' raised:
+    # LoweringException: ImportError: cannot import name 'triton_key' from 'triton.compiler.compiler'
+    "test_fts_dynamo_enforce_p0"
+    "test_fts_dynamo_resume"
+    "test_fts_dynamo_intrafit"
+  ]
+  ++ lib.optionals (pythonAtLeast "3.13") [
+    # RuntimeError: Dynamo is not supported on Python 3.13+
+    "test_fts_dynamo_enforce_p0"
+    "test_fts_dynamo_resume"
+  ]
+  ++ lib.optionals (stdenv.hostPlatform.isAarch64 && stdenv.hostPlatform.isLinux) [
+    # slightly exceeds numerical tolerance on aarch64-linux:
+    "test_fts_frozen_bn_track_running_stats"
+  ];
 
   pythonImportsCheck = [ "finetuning_scheduler" ];
 

